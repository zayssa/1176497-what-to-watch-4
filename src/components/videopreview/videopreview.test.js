import React from "react";
import renderer from "react-test-renderer";
import Videopreview from "./videopreview.jsx";

it(`Render Videopreview`, () => {
  const tree = renderer
    .create(<Videopreview
      preview="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
