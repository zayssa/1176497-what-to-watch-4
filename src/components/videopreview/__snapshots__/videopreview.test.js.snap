// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Videopreview 1`] = `
<video
  muted={true}
  poster="img/avatar.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  style={
    Object {
      "bottom": 0,
      "height": "auto",
      "left": 0,
      "margin": "auto",
      "position": "absolute",
      "right": 0,
      "top": 0,
      "width": "100%",
    }
  }
/>
`;
